storage/delete.go

```diff
@@ -66,6 +69,12 @@ func (c *client) Delete(ctx context.Context, userPubKey, masterKey, fileHash str
 	if err = os.Remove(filepath.Join(userPath, file)); err != nil && !errors.Is(err, os.ErrNotExist) {
 		return errors.Wrapf(err, "failed to remove file %v (%v)", fileHash, filepath.Join(userPath, file))
 	}
+	fileName := fmt.Sprintf("%v:%v%v", masterKey, fileHash, filepath.Ext(file))
+	if c.config.Cdn.AccessKey != "" && c.config.Cdn.URLUpload != "" && c.cdn != nil {
```

Comment:
Let's move this to the separate function as well, as if I understand correctly it's like isCDNAvailable(), then we some extra steps for cdn.
And the same at DeleteUser function.